tests/main/snap-userd-desktop-app-autostart/task.yaml

```diff
@@ -29,12 +30,10 @@ execute: |
     test -e ~/snap/test-snapd-xdg-autostart/current/foo-autostarted
 
     if [[ "$SPREAD_SYSTEM" != ubuntu-14.04-* ]]; then
-        journalctl --flush
-        test "$(journalctl -t foo.desktop --after-cursor=$cursor | wc -l)" -eq 1
+        test "$(journalctl -t foo.desktop --cursor "$cursor" | wc -l)" -gt 0
     else
-        # 14.04 journalctl does not have cursor, neither --identifier support,
-        # nor --flush
-        test "$(journalctl | grep foo.desktop | wc -l)" -eq 1
+        # 14.04 journalctl does not have cursor, neither --identifier support
```

Comment:
Wait, so does 14.04 have or doesn't have cursors? If it doesn't have cursors then what are we doing in all the generic function and test cases above?

Reply:
IIRC @sergiocazzolato mentioned that systemd got a SRU and it currently has cursors. The comment comes from the time I added the test (and there was no cursors support then).

Reply:
Ah, this explains everything. In that case, please update the comment.